tvtk/pyface/ui/wx/scene.py

```diff
@@ -571,6 +571,10 @@ def OnButtonUp(self, event):
     def _closed_fired(self):
         super(Scene, self)._closed_fired()
         self.picker = None
+        # Remove OnPaint handler for PaintEvent, otherwise
+        # OnPaint tries to reset the size of the nonexisting
+        # renderwindow
+        wx.EVT_PAINT(self._vtk_control, None)
```

Comment:
On other hand, this change is meant to silent error message on Linux + wx 2.8.10
https://github.com/enthought/mayavi/pull/346
The same error does not occur on OSX with wxpython 3.0.2, so I have omitted this change in the reimplementation #346